### ISSUE TITLE
eksctl 0.25.0-rc.0 create cluster role missing ec2:DescribeAccountAttributes permission

### ISSUE THREAD

**Issue by stevensu1977** [weaveworks/eksctl#2488]
**What happened?**

I use eksctl 0.25.0-rc.0 ,
```
 eksctl create cluster \
       --name eksworkshop \
       --version 1.16 \
       --managed
```

I deploy nginx and export svc Loadbalancer ,  svc status alwarys pending ,  I drop down found  eks cluster service role eksctl-eksworkshop-cluster-ServiceRole-1PYVYBBRCXYQE have not  AmazonEKSServicePolicy 

```
  Warning  SyncLoadBalancerFailed  2m32s                  service-controller  Error syncing load balancer: failed to ensure load balancer: error creating load balancer: "AccessDenied: User: arn:aws:sts::111111111111:assumed-role/eksctl-eksworkshop-cluster-ServiceRole-1PYVYBBRCXYQE/1595905611586445806 is not authorized to perform: ec2:DescribeAccountAttributes\n\tstatus code: 403, request id: df47c558-d7b9-4227-94f3-de5c243ecb44"
```

**Versions**
Please paste in the output of these commands:
```
$ eksctl version
0.25.0-rc.0
```

**Comment by stevensu1977** [comment]
I check 0.24.0 , default create cluster role include "eksctl-eksworkshop-cluster-PolicyNLB" , but 0.25.rc  not include this policy. 


{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Action": [
                "elasticloadbalancing:*",
                "ec2:CreateSecurityGroup",
                "ec2:Describe*"
            ],
            "Resource": "*",
            "Effect": "Allow"
        }
    ]
}

**Comment by michaelbeaumont** [comment]
Related:
* https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/629
* https://stackoverflow.com/questions/51597410/aws-eks-is-not-authorized-to-perform-iamcreateservicelinkedrole

**Comment by michaelbeaumont** [comment]
@stevensu1977 Are you by any chance using a new AWS account? Can you try creating an NLB then removing it and redeploying the Service?

**Comment by klausenbusk** [comment]
Related: https://github.com/weaveworks/eksctl/issues/2570

**Comment by michaelbeaumont** [comment]
This issue should also be closed by #2571
